brigade-controller/cmd/brigade-controller/main.go

```diff
@@ -31,6 +32,7 @@ func main() {
 	flag.StringVar(&ctrConfig.WorkerImage, "worker-image", defaultWorkerImage(), "kubernetes worker image")
 	flag.StringVar(&ctrConfig.WorkerPullPolicy, "worker-pull-policy", defaultWorkerPullPolicy(), "kubernetes worker image pull policy")
 	flag.StringVar(&ctrConfig.WorkerServiceAccount, "worker-service-account", defaultWorkerServiceAccount(), "kubernetes worker service account name")
+	flag.StringVar(&ctrConfig.JobServiceAccount, "job-service-account", defaultJobServiceAccount(), "kubernetes job service account name")
```

Comment:
This means the service account used by Brigade jobs is a global setting per Brigade deployment - meaning _all_ projects ran by this Brigade instance must use the same service account. 

Would it make sense for this to be a per-project setting? I might want to grant one project the ability to create resources, but not others - for example a CD pipeline that needs to deploy a new version of a deployment - but not others. In the version proposed here, I'd need to either grant all of them this permission, or have two Brigade instances (which is the case right now).

What do you think?

Also, nit: `"kubernetes job service account name"` - since we're not creating actual Kubernetes jobs, this should probably be `"brigade job service account name"`.

Reply:
I actually was going to add that ability in another pr. That way there can be a global default and jobs can override it if needed.

And yes I can change the help text.